pom.xml

```diff
@@ -811,13 +805,13 @@
         <apache.felix.scr.ds.annotations.version>1.2.4</apache.felix.scr.ds.annotations.version>
 
         <!-- Carbon kernel version -->
-        <carbon.kernel.version>4.6.2-m9</carbon.kernel.version>
+        <carbon.kernel.version>4.6.3-m2</carbon.kernel.version>
         <carbon.kernel.feature.version>4.6.1</carbon.kernel.feature.version>
         <carbon.kernel.imp.pkg.version.range>[4.5.0, 5.0.0)</carbon.kernel.imp.pkg.version.range>
         <carbon.kernel.registry.imp.pkg.version.range>[1.0.1, 2.0.0)</carbon.kernel.registry.imp.pkg.version.range>
 
         <!-- Carbon Identity Framework version -->
-        <carbon.identity.framework.version>5.20.42</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.20.104-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
will be updated once https://github.com/wso2/carbon-identity-framework/pull/3585 is merged and a new version is released.